src/content/docs/apm/agents/c-sdk/instrumentation/distributed-tracing-c-sdk.mdx

```diff
@@ -0,0 +1,273 @@
+
+The C SDK requires you to manually instrument your C services, unlike the auto-instrumentation of the other New Relic agents. This means you need to add some lines to your code to use the C SDK, and then you add an additional configuration to enable distributed tracing.
+
+Here's how to get started:
+
+* [Instrument a sample program (recommended):](#quick-start-apm) A fast way to get started
+* [Manual instrumentation:](#manual-instrumentation) Tips for additional instrumentation
+
+<Callout variant="tip">
+  If you want to get more background before getting started, check out these topics:
+  * [How span sampling works](/docs/understand-dependencies/distributed-tracing/get-started/how-new-relic-distributed-tracing-works#sampling) explains distributed tracing options.
+  * [Impacts to APM](/docs/apm/distributed-tracing/getting-started/transition-guide-distributed-tracing) tells you what to expect if you are a current APM user but haven't set up distributed tracing.
+</Callout>
+
+## Instrument a sample program (recommended): [#quick-start-apm]
+
+This is a quick way to see how to install the C SDK and instrument a service with [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based). 
+
+<Callout variant="tip">
+The advanced option called Infinite Tracing isn't available with C SDK.
+</Callout>
+
+You'll need a New Relic account to set up distributed tracing. If you don't already have one, you can quickly [create a free account](https://newrelic.com/signup). Note that this link will take you to another site to complete the sign-up, but you can return here and follow the setup steps below:
+
+### Step 1. Identify services
+
+Figure out which services touch your request so you can instrument each of them to send trace data to New Relic.
+
+### Step 2. Instrument each service with an APM agent [#update-agents]
+
+We have an example of how to install the C SDK and instrument a sample app so it begins to report telemetry to New Relic. Once you understand how it works, you can apply the principles to each of your C services that are involved in your requests. If some of your services use other languages, simply follow the [installation steps](/docs/distributed-tracing/enable-configure/quick-start) for those languages.
+
+Before going to the sample service below, note that it will have you create a default [`newrelic_app_config_t`](https://newrelic.github.io/c-sdk/structnewrelic__app__config__t.html), but you'll also need to add `distributed_tracing.enabled` and set it to `true`:
```

Comment:
This is a valid c-file, but it needs to have the context of the rest of the repo as well.
Additionally, the makefile in the repo is important to run and link the libraries.
Link to example on how to do DT in c-sdk: https://github.com/newrelic/c-dt-example/blob/master/client/client.c
The makefile is really what makes it easy for them: https://github.com/newrelic/c-dt-example/blob/master/client/Makefile

So have the c code here, but definitely link to the repo for the wider context.